modules/apps/transfer/keeper/relay.go

```diff
@@ -243,9 +242,6 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, data t
 
 			// send to receiver
 			moduleAddr := k.authKeeper.GetModuleAddress(types.ModuleName)
-			if moduleAddr == nil {
```

Comment:
I guess if it is nil it will fail on `SendCoins` below, right? Is that the reasoning to remove this?